File: engines/plot/bemf1_graphics_electrodes_simple_new.m
function [ ] = bemf1_graphics_electrodes_simple(P, t, strge, IndicatorElectrodes)
    for m = 1:strge.NumberOfElectrodes    
        p = patch('vertices', P, 'faces', t(IndicatorElectrodes==m, :)); 
        p.FaceColor = strge.Color(m,:);
        p.EdgeColor = 'k';
        p.FaceAlpha = 1.0;
        vector      = strge.PositionOfElectrodes(m, :) + 10*strge.PositionOfElectrodes(m, :)/norm(strge.PositionOfElectrodes(m, :));    
        text(vector(1), vector(2), vector(3), num2str(m), 'color', 'w', 'fontsize', 15);
    end
end

File: engines/plot/bemf1_graphics_electrodes_simple_new.py
import numpy as np
from vedo import Mesh, Text3D, show

def bemf1_graphics_electrodes_simple(P, t, strge, IndicatorElectrodes):
    actors = []
    for m in range(strge.NumberOfElectrodes):
        p = Mesh([P, t[IndicatorElectrodes == m, :]])
        p.color(strge.Color[m])
        p.linecolor("k")
        p.alpha(1.0)
        vector = (strge.PositionOfElectrodes[m, :] + 10 * 
                  strge.PositionOfElectrodes[m, :] / 
                  np.linalg.norm(strge.PositionOfElectrodes[m, :]))
        label = Text3D(str(m), pos = vector, s = 15, c = "w")
        actors.append(p)
        actors.append(label)
    show(*actors)